src/components/grid-aware/BlockQuoteBlock/BlockQuoteBlock.module.css

```diff
@@ -0,0 +1,101 @@
+/* Outter Grid */
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 30px 1fr 30px;
+    grid-template-rows: auto;
+  }
+}
+
+@media (--tablet-and-down) {
+  .bleedBackground {
+    grid-row: 1;
+    grid-template-rows: auto;
+  }
+}
+
+/* Main grid */
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-column-start: 2;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+  padding: 100px 0;
```

Comment:
You should get rid of this padding, since it's not actually present in the design. Adding padding like this will make it harder to reuse the component, since this particular element surrounds the element that actually has the gray background, so this is portion is "invisible". By putting padding here, it functions more like "margin", at least in the sense of putting invisible spacing between the gray box and other components on the page. This means that you could never place this component any closer than 100px away from another component on the page.

In order to ensure that we can reuse this component in as many contexts as possible, we generally should not put any margin in the component definition itself. Here, even though it is padding and not margin, because it is padding in the transparent area it basically functions as margin, since it's the gray box that a person will perceive to be the component itself.

Reply:
👍 Deleted padding.